app/models/asset.rb

```diff
@@ -159,12 +160,15 @@ class Asset < Kithe::Asset
   # Our DziFiles object to manage associated DZI (deep zoom, for OpenSeadragon
   # panning/zooming) file(s).
   #
-  #     asset.dzi_file.url # url to manifest file
-  #     asset.dzi_file.exists?
-  #     asset.dzi_file.create # normally handled by automatic lifecycle hooks
-  #     asset.dzi_file.delete # normally handled by automatic lifecycle hooks
-  def dzi_file
-    @dzi_file ||= DziFiles.new(self)
+  #     asset.dzi_package.url # url to manifest file
+  #     asset.dzi_package.exists?
+  #     asset.dzi_package.create # normally handled by automatic lifecycle hooks
+  #     asset.dzi_package.delete # normally handled by automatic lifecycle hooks
+  #
+  # See also dzi_manifest_file which points to the single .dzi file -- this object
+  # manages a whole directory.
+  def dzi_package
+    @dzi_files ||= DziFiles.new(self)
```

Comment:
Why not memoize with the same variable name:
`@dzi_package ||= DziFiles.new(self)`

Reply:
good catch, just something I didn't notice since tests passed without it, will do.